### ISSUE TITLE
Seth support for Ledger browser support removal

### ISSUE THREAD

**Issue by wolflo** [dapphub/dapptools#119]
Ledger started enabling browser support by default: https://support.ledger.com/hc/en-us/articles/360004981213-Browser-support-removal. 

More specifically, it looks like they removed the browser support setting entirely, leaving no option to turn it off. Trying to use a ledger wallet with updated firmware with Seth results in the error: 

`ethsign: couldn't use Ledger: needs to be in Ethereum app with browser support off`.

**Comment by weijiekoh** [comment]
+1. This is a blocker for using the Ledger via CLI. The non-CLI workaround is to use MetaMask + MyCrypto.

**Comment by asymmetric** [comment]
Link is dead, is there an announcement somewhere?

**Comment by nward13** [comment]
I can't find a new version of the support page, but I just got it working. Turns out I needed to add some udev rules. Some info on how to do this is at https://support.ledger.com/hc/en-us/articles/115005165269-Fix-connection-issues under the "Linux" dropdown. @weijiekoh 

For posterity, Ledger Nano S was working with Ledger Live and other desktop apps but not with Seth (saying browser support needed to be turned off), My Ether Wallet, or My Crypto web apps (both giving a Transport Error, Error Code 5, or some variation on needing to turn browser support on). Adding udev rules appears to have fixed them all.

@asymmetric thanks for diving into a false alarm for me.

**Comment by weijiekoh** [comment]
> but not with Seth (saying browser support needed to be turned off)

Yeah, this issue still persists even with the udev rules.

**Comment by OdysLam** [comment]
According to the ledger documentation and what I am seeing as a Ledgert user, I think the canonical way is for the app to communicate via websockets with ledger live and let the ledger live app do the talking with the device. 

They offer libraries in JS, it shouldn't be too hard to create a bridging script so that ethsign talks with ledger-live. 

What do you all think ? 

https://github.com/LedgerHQ/ledger-live-common/blob/master/docs/gist-tx.md

**Comment by d-xo** [comment]
I think that requiring a running instance of ledger live to use seth would feel quite clunky.

What would the advantage be? Are you running into this issue with the browser support @OdysLam?

**Comment by OdysLam** [comment]
Hey @d-xo, 

I was under the impression from this thread that ethesign does not currently support ledger. I tried using it with my ledger device on a mac, but couldn't figure out the browser support config. 

I will retry and see to add anything to the docs for posterity

**Comment by MrChico** [comment]
`ethsign` works with ledger. There's no browser involved so you can just communicate with the ledger directly. What problem are you seeing?

**Comment by OdysLam** [comment]
I tried using the ledger but it could not find the account. Let me try again to see if I missed any configuration.
